src/media-segment-request.js

```diff
@@ -269,9 +269,13 @@ const transmuxAndNotify = ({
       });
       trackInfoFn = null;
 
-      audioStartFn(probeResult.audioStart);
+      if (probeResult.hasAudio) {
```

Comment:
This seems more like a fix/different pull request

Reply:
yeah, this and another change I have can/should go in another PR, I'll make sure to remove it when I'm fully done with the changes I want to make for this.